src/components/index.ts

```diff
@@ -0,0 +1,2 @@
+// local component
+export { DatePicker } from './DatePicker'
```

Comment:
This is here so that Storybook can find the "local" DatePicker component. The app needs to do the same thing (but there, DatePicker will be an external component).

This is because Storybook can't find `@bcrs-shared-components/DatePicker` ... I don't know why. Lots of time spent on this already :(  This works and it's not terribly ugly.